controller/projects.go

```diff
@@ -136,9 +153,9 @@ func (p *Projects) createInitialAccounts(
 
 		account.SetState(make(model.AccountState))
 
-		if i < len(initialContracts) {
+		/*if i < len(initialContracts) {
```

Comment:
These comments can be deleted now I think.

Reply:
yes